### PR TITLE
Azure CLI documentation

### PR DESCRIPTION
**Pull request by joesiewert**

### CODE REVIEW

File: _basic/continuous-deployment/deployment-with-azurecli.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Deployment With Azure CLI
+menus:
+  basic/cd:
+    title: Azure CLI
+    weight: 7
```
Comment (ethangj) [inline _basic/continuous-deployment/deployment-with-azurecli.md]: Is the weight right? Both these articles are set to weight 7.

File: _basic/continuous-deployment/deployment-with-azurecli.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Deployments to [Azure](https://azure.microsoft.com/en-us) can be done using the Azure CLI directly on a Codeship Basic build machine similar to how you might use the CLI locally.
+
+You will need to install the CLI, configure your authentication with environment variables and then define the CLI commands you want to run as a [custom-script deployment pipeline]({{ site.baseurl }}{% link _basic/continuous-deployment/deployment-with-custom-scripts.md %}).
+
+### Installing The CLI
+
+The [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/overview?view=azure-cli-latest) _does not_ come pre-installed on Codeship Basic build machines.
+
+Please add the following command in the Setup Commands section of your test settings to install the Azure CLI:
```
Comment (ethangj) [inline _basic/continuous-deployment/deployment-with-azurecli.md]: Can we link to the Basic quickstart article anytime we mention setup commands?

File: _basic/continuous-deployment/deployment-with-azurecli.md (hunk 1 above)
Comment (joesiewert) [reply _basic/continuous-deployment/deployment-with-azurecli.md]: Getting a separate PR ready to adjust all the weights. 🏋️

File: _basic/continuous-deployment/deployment-with-azurecli.md (hunk 2 above)
Comment (joesiewert) [reply _basic/continuous-deployment/deployment-with-azurecli.md]: Sure, I can do a separate PR to make that change on all the CLI docs (AWS, Google, Azure).

**Comment by ethangj** [comment]
@joesiewert sounds good, approved!

**Pull request merged**